src/schedule.py

```diff
@@ -58,20 +58,20 @@ def getEventType(schedule, event_date, city):
 
 
 def createEvents(year, schedule):
-    #my_astral.solar_depression = 'civil'
-    # city = a['Buffalo']     # close enuff, about 5 minutes behind ROC in sunset times
-    # city = a['Rochester']     # custom addition: Rochester,USA,43°10'N,77°36'W,US/Eastern,153
-    city = LocationInfo('Rochester_HoP', region='USA', timezone='US/Eastern', latitude=43.1606355, longitude=-77.3883843) # custom addition: Rochester_HoP,USA,43°09'N,77°23'W,US/Eastern,170
+    '''create events from a schedule object at a custom location'''
+    # custom observer: Rochester_HoP,USA,43°09'N,77°23'W,US/Eastern,170
+    observer_location = Observer(latitude=43.1606355, longitude=-77.3883843, elevation=170)
+    tz = timezone('US/Eastern')

```

Comment:
This is good. Eventually we can roll these functions into a class, allowing caller to feed custom observer and tz.

Reply:
Also, **observer_location** can just be named **observer**.

Reply:
because the Class name was observer, I opted for a more verbose name. I am okay with observer.

Reply:
I think astral.Observer should have a timezone member. Maybe we can add an issue to astral

Reply:
> I think astral.Observer should have a timezone member. Maybe we can add an issue to astral

I had a similar thought, but looking over their **__main__.py** example and **sun.py**, I don't think they'd go for that because it looks like **LocationInfo** is intended to take a TZ as an arg, while **Observer** is a representation of a time-agnostic location.  All the functions in their **sun** module take an Observer, a date, and a tzinfo already.  The change would be API-breaking.